src/assets/stylesheets/index.scss

```diff
@@ -673,618 +673,6 @@ section.navbar-fixed {
   overflow: hidden;
 }
 
-/* navbar
-=================================================================== */
-
-.navbar {
-  padding: 0;
-  min-height: 75px;
-  box-shadow: 0 0 3px rgba(0, 0, 0, 0.2);
-}
-.navbar-brand {
-  /* background: $logo-dark no-repeat center center / contain; */
-  width: 120px;
-  @extend .text-hide;
-  height: 75px;
-  padding: 25px 0;
-}
-.navbar-brand img {
-  margin: 0;
-}
-
-.nav {
-  margin: 0;
-}
-.nav.navbar-right {
-  margin-left: auto;
-  margin-right: 45px;
-}
-
-.nav > li,
-.nav > li > a {
-  position: relative;
-  display: block;
-}
```

Comment:
If I'm not mistaken the following part got lost:

```scss
.nav > li > a {	
  position: relative;	
  display: block;	
}
```

Reply:
I think this part is not needed, it only breaks nav items which have arrows but without it everything looks fine for me when files are imported in the proper order. Hopely I am not missing something here.